python/dask_cudf/dask_cudf/tests/utils.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import numpy as np
+import pandas as pd
+
+import dask.dataframe as dd
+
+import cudf
+
+
+def _make_empty_frame(npartitions=2):
+    df = pd.DataFrame({"x": [], "y": []})
+    gdf = cudf.DataFrame.from_pandas(df)
+    dgf = dd.from_pandas(gdf, npartitions=npartitions)
+    return dgf
```

Comment:
Seems strange that this only returns `dgf` while `_make_random_frame` and `_make_random_frame_float` return `df, dgf`. Should we symmetrize this?

Reply:
Sorry, I probably shouldn't have moved this function in the first place since it's being consumed elsewhere and not actually used in my tests. I just moved it back for now.